doc/kubeletConfig-check.md

```diff
@@ -0,0 +1,141 @@
+# KubeletConfig default checks usage
+
+This page describes the usage of the `KubeletConfig` default checks in the compliance operator.
+`KubeletConfig` rules are being changed from Node checks to Platform checks. Compliance Operator
+is able to check runtime `KubeletConfig` through `Node/Proxy` object, and it is able to check `KubeletConfig`
+configurations passed in as command flags and from other sources than `/etc/kubernetes/kubelet.conf`.
+It provides better accuracy in checking `KubeletConfig` configurations.
+
+## How does it work?
+To give an example on the CIS benchmark:
```

Comment:
I might jump into how things work today. I'm wondering if adding this might confuse users who are looking to get right in the meat of the feature.